prover/src/zkevm/circuit.rs

```diff
@@ -23,10 +23,10 @@ pub use self::builder::{
 pub const MAX_TXS: usize = 100;
 pub const MAX_INNER_BLOCKS: usize = 100;
 pub const MAX_EXP_STEPS: usize = 10_000;
-pub const MAX_CALLDATA: usize = 400_000;
-pub const MAX_BYTECODE: usize = 400_000;
-pub const MAX_MPT_ROWS: usize = 400_000;
-pub const MAX_KECCAK_ROWS: usize = 524_000;
+pub const MAX_CALLDATA: usize = 600_000;
+pub const MAX_BYTECODE: usize = 600_000;
+pub const MAX_MPT_ROWS: usize = 1_000_000;
+pub const MAX_KECCAK_ROWS: usize = 1_000_000;
```

Comment:
this limit shares with Poseidon circuit too?

Reply:
poseidon uses MAX_MPT_ROWS now.. (historical tech debt. will refactor later)